File: mockez-back/src/main/java/com/mockez/service/TableService.java
package com.mockez.service;

import com.mockez.domain.model.entity.Table;

import java.util.List;
import java.util.UUID;

/**
 * @author Luong Quoc Trung, Do Quoc Viet
 */

public interface TableService {

    Table getTable(UUID id);

    List<Table> getTablesBySchema(UUID schemaId);

    UUID saveOrUpdateTable(Table table);
}

File: mocker-front/src/app/core/service/table.service.ts
import { Injectable } from '@angular/core';
import { TableModel } from '@core/domain/model/entity/table.model';
import { AbstractService } from '@core/service/abstract.service';

/**
 * @author Do Quoc Viet
 */

@Injectable({
  providedIn: 'root'
})
export class TableService extends AbstractService<TableModel> {
  override ENTITY_URL: string = 'table';
}
